doc/conf.py

```diff
@@ -346,7 +346,9 @@
 # Error when parsing function declaration and more.  This is a list
 # of strings that the parser additionally should accept as
 # attributes.
-cpp_id_attributes = ['__syscall', '__syscall_inline', '__deprecated' ]
+cpp_id_attributes = ['__syscall', '__syscall_inline', '__deprecated',
+    '__may_alias', '__used', '__unused', '__weak',
+    '__DEPRECATED_MACRO', 'CODE_UNREACHABLE', 'FUNC_NORETURN' ]
```

Comment:
I think you may not need CODE_UNREACHABLE, it is only used in the body of functions.

Reply:
IT VANISHES IN A CLOUD OF GREASY BLACK SMOKE